### PR TITLE
IPv4 location markup batch #2

### PR DESCRIPTION
**Pull request by Yullia**
IPv4 location markup batch 2 (50-100).

### CODE REVIEW

File: meta/7f0408de.csv [hunk 1]
```diff
@@ -49,26 +49,26 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
...
+24996,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,131:131,F,F,133,143,F,F,,,,,0,0,F,F,F,Authentication Credentials
+24998,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,133:133,F,F,124,134,F,F,,,,,0,0,F,F,F,Authentication Credentials
+25001,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,134:134,F,F,133,143,F,F,,,,,0,0,F,F,F,Authentication Credentials
+25002,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,136:136,F,F,124,134,F,F,,,,,0,0,F,F,F,Authentication Credentials
+25003,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,137:137,F,F,133,143,F,F,,,,,0,0,F,F,F,Authentication Credentials
+25004,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,140:140,F,F,124,134,F,F,,,,,0,0,F,F,F,Authentication Credentials
+25005,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,141:141,F,F,133,143,F,F,,,,,0,0,F,F,F,Authentication Credentials
+25006,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,143:143,F,F,124,134,F,F,,,,,0,0,F,F,F,Authentication Credentials
+25007,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,144:144,F,F,133,143,F,F,,,,,0,0,F,F,F,Authentication Credentials
```
Comment (babenek) [inline meta/7f0408de.csv:71]:
```suggestion
24627,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,119:119,T,F,133,143,F,F,,,Info,,0,0,F,F,F,Other
24628,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,118:118,T,F,124,134,F,F,,,Info,,0,0,F,F,F,Other
24629,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,116:116,T,F,133,143,F,F,,,Info,,0,0,F,F,F,Other
24630,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,115:115,T,F,124,134,F,F,,,Info,,0,0,F,F,F,Other
24677,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,121:121,T,F,124,134,F,F,,,Info,,0,0,F,F,F,Other
24816,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,122:122,T,F,133,143,F,F,,,Info,,0,0,F,F,F,Other
24817,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,124:124,T,F,124,134,F,F,,,Info,,0,0,F,F,F,Other
24818,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,125:125,T,F,133,143,F,F,,,Info,,0,0,F,F,F,Other
24993,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,127:127,T,F,124,134,F,F,,,Info,,0,0,F,F,F,Other
24994,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,128:128,T,F,133,143,F,F,,,Info,,0,0,F,F,F,Other
24995,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,130:130,T,F,124,134,F,F,,,Info,,0,0,F,F,F,Other
24996,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,131:131,T,F,133,143,F,F,,,Info,,0,0,F,F,F,Other
24998,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,133:133,T,F,124,134,F,F,,,Info,,0,0,F,F,F,Other
25001,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,134:134,T,F,133,143,F,F,,,Info,,0,0,F,F,F,Other
25002,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,136:136,T,F,124,134,F,F,,,Info,,0,0,F,F,F,Other
25003,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,137:137,T,F,133,143,F,F,,,Info,,0,0,F,F,F,Other
25004,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,140:140,T,F,124,134,F,F,,,Info,,0,0,F,F,F,Other
25005,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,141:141,T,F,133,143,F,F,,,Info,,0,0,F,F,F,Other
25006,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,143:143,T,F,124,134,F,F,,,Info,,0,0,F,F,F,Other
25007,67e9fdda,GitHub,7f0408de,data/7f0408de/other/67e9fdda.md,144:144,T,F,133,143,F,F,,,Info,,0,0,F,F,F,Other
```

**Review by babenek — DISMISSED** [review]
LGTM

File: meta/77a3d7d7.csv [hunk 2]
```diff
@@ -175,7 +175,7 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 110039,311f58f1,GitHub,77a3d7d7,data/77a3d7d7/test/311f58f1.groovy,229:229,F,F,,,F,F,,,,,0.00,,F,F,F,Generic Secret
 130015,fce6a5c7,GitHub,77a3d7d7,data/77a3d7d7/src/fce6a5c7.java,402:402,F,F,,,F,F,,,,,0.00,,F,F,F,Generic Secret
 131901,a82445ea,GitHub,77a3d7d7,data/77a3d7d7/test/a82445ea.groovy,20:20,T,F,28,147,F,F,,,,,0.00,,F,F,F,Predefined Pattern
-132041,06572a96,GitHub,77a3d7d7,data/77a3d7d7/other/06572a96.md,23:23,F,F,,,F,F,,,,,0.00,,F,F,F,Other
+132041,06572a96,GitHub,77a3d7d7,data/77a3d7d7/other/06572a96.md,23:23,F,F,62,70,F,F,,,,,0.00,,F,F,F,Other
```
Comment (Yullia) [inline meta/77a3d7d7.csv:178]:
```suggestion
132041,06572a96,GitHub,77a3d7d7,data/77a3d7d7/other/06572a96.md,23:23,F,F,62,70,F,F,,,Info,,0.00,,F,F,F,Other
```

File: meta/255bae6f.csv [hunk 3]
```diff
@@ -317,7 +317,7 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 131463,811a6971,GitHub,255bae6f,data/255bae6f/other/811a6971,138:138,F,F,,,F,F,,,,,0.00,,F,F,F,Password
 131464,353417a6,GitHub,255bae6f,data/255bae6f/src/353417a6.rb,94:94,F,F,,,F,F,,,,,0.00,,F,F,F,Password
 131469,811a6971,GitHub,255bae6f,data/255bae6f/other/811a6971,141:141,F,F,,,F,F,,,,,0.00,,F,F,F,Password
-131955,d09ea66f,GitHub,255bae6f,data/255bae6f/other/d09ea66f,121:121,F,F,,,F,F,,,,,0.00,,F,F,F,Other
+131955,d09ea66f,GitHub,255bae6f,data/255bae6f/other/d09ea66f,121:121,F,F,17,24,F,F,,,,,0.00,,F,F,F,Other
```
Comment (Yullia) [inline meta/255bae6f.csv:320]:
```suggestion
131955,d09ea66f,GitHub,255bae6f,data/255bae6f/other/d09ea66f,121:121,F,F,17,24,F,F,,,Info,,0.00,,F,F,F,Other
```

File: meta/850c2319.csv [hunk 4]
```diff
@@ -180,7 +180,7 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 130482,999981ff,GitHub,850c2319,data/850c2319/test/999981ff.go,27:27,T,F,19,21,F,F,Any,,,Secret,1.00,2,F,F,F,Password
 130483,5ac4f856,GitHub,850c2319,data/850c2319/other/5ac4f856.md,21:21,T,F,11,13,F,F,Any,,,Secret,1.00,2,F,F,F,Password
 130484,128a3f9f,GitHub,850c2319,data/850c2319/other/128a3f9f.md,15:15,T,F,10,12,F,F,Any,,,Secret,1.00,2,F,F,F,Password
-132091,eca158e4,GitHub,850c2319,data/850c2319/src/eca158e4.go,314:314,F,F,,,F,F,,,,,0.00,,F,F,F,Other
+132091,eca158e4,GitHub,850c2319,data/850c2319/src/eca158e4.go,314:314,F,F,90,97,F,F,,,,,0.00,,F,F,F,Other
```
Comment (Yullia) [inline meta/850c2319.csv:183]:
```suggestion
132091,eca158e4,GitHub,850c2319,data/850c2319/src/eca158e4.go,314:314,F,F,90,97,F,F,,,Info,,0.00,,F,F,F,Other
```

File: meta/894e3377.csv [hunk 5]
```diff
@@ -719,37 +719,37 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
...
+132109,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,356:356,F,F,24,32,F,F,,,,,0.00,,F,F,F,Other
+132110,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,357:357,F,F,24,32,F,F,,,,,0.00,,F,F,F,Other
+132111,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,358:358,F,F,24,31,F,F,,,,,0.00,,F,F,F,Other
+132112,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,359:359,F,F,25,32,F,F,,,,,0.00,,F,F,F,Other
+132113,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,360:360,F,F,22,30,F,F,,,,,0.00,,F,F,F,Other
+132114,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,361:361,F,F,23,31,F,F,,,,,0.00,,F,F,F,Other
+132115,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,362:362,F,F,26,33,F,F,,,,,0.00,,F,F,F,Other
+132116,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,363:363,F,F,29,37,F,F,,,,,0.00,,F,F,F,Other
+132117,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,364:364,F,F,17,25,F,F,,,,,0.00,,F,F,F,Other
+132118,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,365:365,F,F,26,34,F,F,,,,,0.00,,F,F,F,Other
```
Comment (Yullia) [inline meta/894e3377.csv:742]:
```suggestion
132098,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,345:345,F,F,17,25,F,F,,,Info,,0.00,,F,F,F,Other
132099,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,346:346,F,F,20,27,F,F,,,Info,,0.00,,F,F,F,Other
132100,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,347:347,F,F,22,30,F,F,,,Info,,0.00,,F,F,F,Other
132101,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,348:348,F,F,21,29,F,F,,,Info,,0.00,,F,F,F,Other
132102,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,349:349,F,F,32,40,F,F,,,Info,,0.00,,F,F,F,Other
132103,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,350:350,F,F,23,30,F,F,,,Info,,0.00,,F,F,F,Other
132104,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,351:351,F,F,25,32,F,F,,,Info,,0.00,,F,F,F,Other
132105,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,352:352,F,F,32,39,F,F,,,Info,,0.00,,F,F,F,Other
132106,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,353:353,F,F,29,37,F,F,,,Info,,0.00,,F,F,F,Other
132107,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,354:354,F,F,35,43,F,F,,,Info,,0.00,,F,F,F,Other
132108,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,355:355,F,F,18,26,F,F,,,Info,,0.00,,F,F,F,Other
132109,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,356:356,F,F,24,32,F,F,,,Info,,0.00,,F,F,F,Other
132110,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,357:357,F,F,24,32,F,F,,,Info,,0.00,,F,F,F,Other
132111,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,358:358,F,F,24,31,F,F,,,Info,,0.00,,F,F,F,Other
132112,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,359:359,F,F,25,32,F,F,,,Info,,0.00,,F,F,F,Other
132113,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,360:360,F,F,22,30,F,F,,,Info,,0.00,,F,F,F,Other
132114,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,361:361,F,F,23,31,F,F,,,Info,,0.00,,F,F,F,Other
132115,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,362:362,F,F,26,33,F,F,,,Info,,0.00,,F,F,F,Other
132116,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,363:363,F,F,29,37,F,F,,,Info,,0.00,,F,F,F,Other
132117,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,364:364,F,F,17,25,F,F,,,Info,,0.00,,F,F,F,Other
132118,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,365:365,F,F,26,34,F,F,,,Info,,0.00,,F,F,F,Other
```

File: meta/894e3377.csv [hunk 6]
```diff
@@ -719,37 +719,37 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
...
-132125,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,380:380,F,F,,,F,F,,,,,0.00,,F,F,F,Other
-132126,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,381:381,F,F,,,F,F,,,,,0.00,,F,F,F,Other
-132127,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,382:382,F,F,,,F,F,,,,,0.00,,F,F,F,Other
-132128,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,383:383,F,F,,,F,F,,,,,0.00,,F,F,F,Other
+132123,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,375:375,F,F,32,41,F,F,,,,,0.00,,F,F,F,Other
+132124,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,378:378,F,F,27,36,F,F,,,,,0.00,,F,F,F,Other
+132125,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,380:380,F,F,27,36,F,F,,,,,0.00,,F,F,F,Other
+132126,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,381:381,F,F,26,35,F,F,,,,,0.00,,F,F,F,Other
+132127,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,382:382,F,F,39,47,F,F,,,,,0.00,,F,F,F,Other
+132128,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,383:383,F,F,29,38,F,F,,,,,0.00,,F,F,F,Other
```
Comment (Yullia) [inline meta/894e3377.csv:752]:
```suggestion
132123,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,375:375,F,F,32,41,F,F,,,Info,,0.00,,F,F,F,Other
132124,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,378:378,F,F,27,36,F,F,,,Info,,0.00,,F,F,F,Other
132125,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,380:380,F,F,27,36,F,F,,,Info,,0.00,,F,F,F,Other
132126,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,381:381,F,F,26,35,F,F,,,Info,,0.00,,F,F,F,Other
132127,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,382:382,F,F,39,47,F,F,,,Info,,0.00,,F,F,F,Other
132128,ed108d6d,GitHub,894e3377,data/894e3377/src/ed108d6d.php,383:383,F,F,29,38,F,F,,,Info,,0.00,,F,F,F,Other
```

**Review by babenek — APPROVED** [review]
LGTM

**Review by iuriimet — DISMISSED** [review]

**Review by babenek — APPROVED** [review]
merged well. LGTM

**Review by xDizzix — APPROVED** [review]

**Pull request merged**